### PR TITLE
adding imaging schemas

### PR DESCRIPTION
**Pull request by dyf**
Schemas for `Acquisition` and `Instrument`.

I created a base `Device` that is reused for everything with a model/manufacturer/serial#. Otherwise no other major changes.

### CODE REVIEW

**Comment by adamkglaser** [comment]
Amazing, this looks good to me.

File: src/aind_data_schema/imaging/acquisition.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    )
+    voxel_size: float = Field(..., title='Voxel size (um)')
+    volume_size: float = Field(
+        ...,
+        description='Size of the volume for this dimension.',
+        title='Volume size (um)',
+    )
+
+
+class Laser(BaseModel):
```
Comment (saskiad) [inline src/aind_data_schema/imaging/acquisition.py:60]: there's a bunch of other fields for the laser that we moved into acquisition - I think this is based on outdated version

Comment (dyf) [reply src/aind_data_schema/imaging/acquisition.py:60]: I think this matches the latest JSON schemas.  If they need changing let me know and I will re-generate this.

**Comment by saskiad** [comment]
> I created a base Device that is reused for everything with a model/manufacturer/serial#. Otherwise no other major changes.

There was already a device object for additional devices - I don't see that in here.

File: src/aind_data_schema/imaging/acquisition.py (hunk 1 above)
Comment (saskiad) [reply src/aind_data_schema/imaging/acquisition.py:60]: this definitely isn't the latest JSON schema - have you had a chance to figure out what happened?

**Review by saskiad — APPROVED** [review]
I'll fix things when it's merged.

**Comment by dyf** [comment]
> There was already a device object for additional devices - I don't see that in here.

Ah, that's what it was.  I repurposed that `Device` for all the other objects with the same device properties.

**Pull request merged**